frontend/components/GravePageAssets.tsx

```diff
@@ -4,39 +4,52 @@ import { useContext, useEffect, useState } from 'react';
 import { getGraveVaultFor } from '@/utils/universalProfile';
 import { Text } from '@chakra-ui/react';
 import { WalletContext } from '@/components/wallet/WalletContext';
+import { getUpAddressUrds } from '@/utils/urdUtils';
 
 export default function GravePageAssets({
-  pageAccount,
-  pageGraveVault = null,
+  graveOwner,
 }: {
-  pageAccount: string;
-  pageGraveVault?: string | null;
+  graveOwner: string;
 }) {
   const walletContext = useContext(WalletContext);
   const { networkConfig } = walletContext;
-  const [graveVault, setGraveVault] = useState<string | null>(pageGraveVault);
+  const [graveVault, setGraveVault] = useState<string | null>(null);
```

Comment:
should we worry about having it as null as the initial state?

Reply:
no because if it's loading (null) the code handles it for the UI